rule.raml

```diff
@@ -0,0 +1,364 @@
+                        "if": ["oic.if.s"],
+                        "ins": "f33a89b2"
+                      },
+                      {
+                        "href": "scenevalue",
+                        "rel": ["boundto"],
+                        "bind": "push",
+                        "anchor": "ocf://5c68f373-0c4c-4604-9922-db639e20533d/heatcontrol",
+                        "if": ["oic.if.a"],
+                        "ins": "ca5f1004"
+                      },
+                      {
+                        "href": "377fac40",
+                        "rt": "oic.r.linkstatus",
+                        "if": ["oic.if.baseline", "oic.if.rw"]
+                      },
+                      {
+                        "href": "f33a89b2",
+                        "rt": "oic.r.linkstatus",
+                        "if": ["oic.if.baseline", "oic.if.rw"]
+                      },
+                      {
+                        "href": "ca5f1004",
+                        "rt": "oic.r.linkstatus",
+                        "if": ["oic.if.baseline", "oic.if.rw"]
+                      }
+                    ]
+
+
+/RuleURI?if=oic.if.rw:
+    description: |
+      Read/write representation of a rule contains the rule string
+    displayName: rule-rw
+
+    get:
+        description: |
+          Read/write interface of the rule resource type
+        responses:
+          200:
+            body:
+              application/json:
+                schema: readwrite
+                  example: |
+                    {
+                      "rule": "(switch:value = true) and (temperature:temperature >= set-temp:temperature)",
+                      "ruleresult": false,
+                      "ruleenable": false,
+                      "actionenable": false
+                    }
+
+    post:
+      body:
+        application/json:
+          schema: rule-rw
+            example: |
+              {
+                "rule": "(switch:value = true) and (temperature:temperature >= set-temp:temperature)"
+              }
+      responses:
+        200:
+          body:
+            application/json:
+              schema: rule-rw
+                example: |
+                  {
+                    "rule": "(switch:value = true) and (temperature:temperature >= set-temp:temperature)",
+                    "ruleresult": false,
+                    "ruleenable": false,
+                    "actionenable": false
+                  }
+
+
+
+/RuleURI?if=oic.if.b:
+    description: |
+      Batch representation of a rule contains the rule string, rule inputs and rule actions
+    displayName: rule-batch
+
+    get:
+        description: |
+          Batch interface of the rule resource type
+        responses:
+          200:
+            body:
+              application/json:
+                schema: batch
+                  example: |
+                    [
+                      {
+                        "href":"",
```

Comment:
OK, this is here because of the self Link above.  I don't think the "href" should be blank though, right?